WordPress/gutenberg
# Cover Block: Allow alt text

## Description
Adds setting to provide alt text when image element is used for a Cover block.

I don't think we need deprecation for this feature since we already had an empty alt text attribute for image elements.

Fixes #30505.

## How has this been tested?
1. Create a post.
2. Add Cover block.
3. Select an image.
4. Textarea for the alt text should be visible in the sidebar.

## Screenshots 
![CleanShot 2021-07-06 at 20 31 00](https://user-images.githubusercontent.com/240569/124636011-20684900-de99-11eb-960b-fd40fe9fc4e3.png)

## Types of changes
New feature

## Checklist:
